docker/dockerfiles/Dockerfile_commcarehq

```diff
@@ -1,7 +1,24 @@
 FROM dimagi/commcarehq_base:latest
 
+RUN mkdir requirements
+COPY scripts/uninstall-requirements.sh requirements/uninstall-requirements.sh
+COPY requirements/uninstall-requirements.txt requirements/uninstall-requirements.txt
+
+RUN requirements/uninstall-requirements.sh
+
+COPY requirements/requirements.txt /vendor/requirements.txt
+COPY requirements/dev-requirements.txt /vendor/dev-requirements.txt
+COPY requirements/test-requirements.txt /vendor/test-requirements.txt
+
+RUN pip install \
+    -r /vendor/requirements.txt \
+    -r /vendor/dev-requirements.txt \
+    --user --upgrade \
+    && rm -rf /root/.cache/pip
```

Comment:
Why remove the cache? (And why not put the cache somewhere outside the container where it will be preserved?) As I understand it, this cache is a place where downloaded files are stored to prevent future redundant downloads. It does not affect installed packages in any way.